cypress/platform/marker_unique_id.html

```diff
@@ -0,0 +1,28 @@
+<html>
+  <head> </head>
+  <body>
+    <h1>Example</h1>
+    <pre class="mermaid">
+      %%{init:{"theme":"base", "themeVariables": {"lineColor":"red"}}}%%
+      flowchart LR
+      subgraph red
+      A --> B
+      end
+    </pre>
+    <pre class="mermaid">
+      %%{init:{"theme":"base", "themeVariables": {"lineColor":"blue"}}}%%
+      flowchart LR
+      subgraph black
+      A --> B
+      end
+    </pre>
```

Comment:
Let's add the new yaml config option to the test. 

```suggestion
    <pre class="mermaid">
      %%{init:{"theme":"base", "themeVariables": {"lineColor":"red"}}}%%
      flowchart LR
      subgraph red
      A --> B
      end
    </pre>
    <pre class="mermaid">
      %%{init:{"theme":"base", "themeVariables": {"lineColor":"blue"}}}%%
      flowchart LR
      subgraph black
      A --> B
      end
    </pre>
    <pre class="mermaid">
      ---
      config:
        theme: base
        themeVariables:
          lineColor: yellow
      ---
      flowchart LR
      subgraph red
      A --> B
      end
    </pre>
    <pre class="mermaid">
      ---
      config:
        theme: base
        themeVariables:
          lineColor: green
      ---
      flowchart LR
      subgraph black
      A --> B
      end
    </pre>
```